### ISSUE TITLE
tests fail with python 3.11

### ISSUE THREAD

**Issue by keszybz** [vasole/pymca#889]
```
======================================================================
ERROR: testSlowRoiFitHdf5Map (PyMcaBatchTest.testPyMcaBatch.testSlowRoiFitHdf5Map)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/builddir/build/BUILD/pymca-5.6.7/PyMca5/tests/PyMcaBatchTest.py", line 67, in tearDown
    super(testPyMcaBatch, self).tearDown()
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/builddir/build/BUILDROOT/PyMca-5.6.7-7.fc37.s390x/usr/lib64/python3.11/site-packages/PyMca5/PyMcaGui/misc/testutils.py", line 250, in tearDown
    if self._currentTestSucceeded():
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/builddir/build/BUILDROOT/PyMca-5.6.7-7.fc37.s390x/usr/lib64/python3.11/site-packages/PyMca5/PyMcaGui/misc/testutils.py", line 203, in _currentTestSucceeded
    self._feedErrorsToResult(result, self._outcome.errors)
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'testPyMcaBatch' object has no attribute '_feedErrorsToResult'
======================================================================
ERROR: testSlowRoiFitSpecMap (PyMcaBatchTest.testPyMcaBatch.testSlowRoiFitSpecMap)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/builddir/build/BUILD/pymca-5.6.7/PyMca5/tests/PyMcaBatchTest.py", line 67, in tearDown
    super(testPyMcaBatch, self).tearDown()
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/builddir/build/BUILDROOT/PyMca-5.6.7-7.fc37.s390x/usr/lib64/python3.11/site-packages/PyMca5/PyMcaGui/misc/testutils.py", line 250, in tearDown
    if self._currentTestSucceeded():
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/builddir/build/BUILDROOT/PyMca-5.6.7-7.fc37.s390x/usr/lib64/python3.11/site-packages/PyMca5/PyMcaGui/misc/testutils.py", line 203, in _currentTestSucceeded
    self._feedErrorsToResult(result, self._outcome.errors)
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'testPyMcaBatch' object has no attribute '_feedErrorsToResult'
======================================================================
ERROR: testSubCommands (PyMcaBatchTest.testPyMcaBatch.testSubCommands)
Check multi processing slicing of 2D maps for different
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/builddir/build/BUILD/pymca-5.6.7/PyMca5/tests/PyMcaBatchTest.py", line 67, in tearDown
    super(testPyMcaBatch, self).tearDown()
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/builddir/build/BUILDROOT/PyMca-5.6.7-7.fc37.s390x/usr/lib64/python3.11/site-packages/PyMca5/PyMcaGui/misc/testutils.py", line 250, in tearDown
    if self._currentTestSucceeded():
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/builddir/build/BUILDROOT/PyMca-5.6.7-7.fc37.s390x/usr/lib64/python3.11/site-packages/PyMca5/PyMcaGui/misc/testutils.py", line 203, in _currentTestSucceeded
    self._feedErrorsToResult(result, self._outcome.errors)
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'testPyMcaBatch' object has no attribute '_feedErrorsToResult'
----------------------------------------------------------------------
Ran 81 tests in 138.992s
```

`_feedErrorsToResult` was removed in commit python/cpython@664448d81f41c5fa971d8523a71b0f19e76cc136

**Comment by vasole** [comment]
I was about to release current master as 5.7.3. Please, can you check if current master passes the tests?

**Comment by vasole** [comment]
Well, not necessary. Tests will not pass.

**Comment by vasole** [comment]
PR #890 should fix the immediate Python 3.11-related problems.

**Comment by keszybz** [comment]
Thanks for the quick resolution!

**Comment by vasole** [comment]
BTW, I released version 5.7.3 containing the changes.
